Cisco/cisco-esa/_meta/smart-descriptions.json

```
@@ -0,0 +1,291 @@
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was delivered"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was delivered",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "delivered"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was delivered"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was quarantined",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "QUARANTINED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was quarantined"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was dropped",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "DROPPED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was dropped"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was bounced",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "BOUNCED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was bounced"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was aborted",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "ABORTED"
```

With the suggestion to lower `event.action`:
```suggestion
                "value": "aborted"
```